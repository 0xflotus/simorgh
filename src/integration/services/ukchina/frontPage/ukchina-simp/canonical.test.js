/**
 * WARNING: DO NOT EDIT. This file is automatically generated by src/integration/utils/runTests/writeTestFiles/writeTestFile.js
 * @service ukchina
 * @pathname /ukchina/simp
 */

import runCanonicalTests from '../../../../pages/frontPage/canonicalTests';

describe('Canonical ukchina frontPage', runCanonicalTests);
