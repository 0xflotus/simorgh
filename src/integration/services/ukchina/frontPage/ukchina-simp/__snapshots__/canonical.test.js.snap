// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Canonical ukchina frontPage A11y I can see the skip to content link 1`] = `"跳过此内容"`;

exports[`Canonical ukchina frontPage Analytics ATI 1`] = `"<img height=\\"1px\\" width=\\"1px\\" alt=\\"\\" style=\\"position:absolute\\" src=\\"https://logws1363.ati-host.net?s=598343&amp;s2=93&amp;p=ukchinasimp.page&amp;x1=[urn:bbc:cps:42868823-bcc9-e9a1-e050-380a4e8044fe]&amp;x2=[responsive]&amp;x3=[news-ukchina]&amp;x4=[zh-Hans]&amp;x7=[index-home]&amp;x8=[simorgh-nojs]&amp;x9=[主页+-+BBC+英伦网]&amp;x11=[2016-12-02T11:14:16.000Z]&amp;x12=[2019-09-08T19:52:54.000Z]\\"/>"`;

exports[`Canonical ukchina frontPage Footer I can click on the BBC branding and it would take me to the homepage 1`] = `"/ukchina"`;

exports[`Canonical ukchina frontPage Footer I can see the footer copyright and external linking text 1`] = `"© 2020 BBC. BBC对外部网站内容不负责任。 阅读了解我们对待外部链接的做法。"`;

exports[`Canonical ukchina frontPage Header I can see the branding 1`] = `"BBC, 英伦网"`;

exports[`Canonical ukchina frontPage I can see Index Alsos 1`] = `"更多相关内容"`;

exports[`Canonical ukchina frontPage SEO Canonical link 1`] = `"http://localhost:7080/ukchina/simp"`;

exports[`Canonical ukchina frontPage SEO Dir attribute 1`] = `"ltr"`;

exports[`Canonical ukchina frontPage SEO FB admins 1`] = `"100004154058350"`;

exports[`Canonical ukchina frontPage SEO FB app ID 1`] = `"1609039196070050"`;

exports[`Canonical ukchina frontPage SEO Heading level 1 1`] = `"BBC, 英伦网 - 主页"`;

exports[`Canonical ukchina frontPage SEO Lang attribute 1`] = `"zh-Hans"`;

exports[`Canonical ukchina frontPage SEO Linked data 1`] = `
Object {
  "@context": "http://schema.org",
  "@type": "WebPage",
  "image": Object {
    "@type": "ImageObject",
    "height": 576,
    "url": "https://news.files.bbci.co.uk/ws/img/logos/og/ukchina.png",
    "width": 1024,
  },
  "mainEntityOfPage": Object {
    "@id": "http://localhost:7080/ukchina/simp",
    "@type": "WebPage",
    "name": "主页",
  },
  "publisher": Object {
    "@type": "NewsMediaOrganization",
    "logo": Object {
      "@type": "ImageObject",
      "height": 576,
      "url": "https://news.files.bbci.co.uk/ws/img/logos/og/ukchina.png",
      "width": 1024,
    },
    "name": "BBC 英伦网",
    "publishingPrinciples": "https://www.bbc.com/ukchina/simp/institutional-51359562",
  },
  "thumbnailUrl": "https://news.files.bbci.co.uk/ws/img/logos/og/ukchina.png",
  "url": "http://localhost:7080/ukchina/simp",
}
`;

exports[`Canonical ukchina frontPage SEO OG description 1`] = `"BBC 英伦网汇集了BBC精选文章，以趣味和知识性的角度向全球华文读者提供关于商业职场、生涯发展、文化艺术、自然环境、未来科技和趋势、旅游探索等不同主题的深度内容。"`;

exports[`Canonical ukchina frontPage SEO OG image 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/ukchina.png"`;

exports[`Canonical ukchina frontPage SEO OG image alt 1`] = `"BBC 英伦网"`;

exports[`Canonical ukchina frontPage SEO OG locale 1`] = `"zh-hans"`;

exports[`Canonical ukchina frontPage SEO OG site name 1`] = `"BBC 英伦网"`;

exports[`Canonical ukchina frontPage SEO OG title 1`] = `"主页 - BBC 英伦网"`;

exports[`Canonical ukchina frontPage SEO OG type 1`] = `"website"`;

exports[`Canonical ukchina frontPage SEO OG url 1`] = `"http://localhost:7080/ukchina/simp"`;

exports[`Canonical ukchina frontPage SEO Page title 1`] = `"主页 - BBC 英伦网"`;

exports[`Canonical ukchina frontPage SEO Robots meta tag 1`] = `"noodp,noydir"`;

exports[`Canonical ukchina frontPage SEO Twitter card 1`] = `"summary_large_image"`;

exports[`Canonical ukchina frontPage SEO Twitter creator 1`] = `"@BBCChina"`;

exports[`Canonical ukchina frontPage SEO Twitter description 1`] = `"BBC 英伦网汇集了BBC精选文章，以趣味和知识性的角度向全球华文读者提供关于商业职场、生涯发展、文化艺术、自然环境、未来科技和趋势、旅游探索等不同主题的深度内容。"`;

exports[`Canonical ukchina frontPage SEO Twitter image alt 1`] = `"BBC 英伦网"`;

exports[`Canonical ukchina frontPage SEO Twitter image src 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/ukchina.png"`;

exports[`Canonical ukchina frontPage SEO Twitter site 1`] = `"@BBCChina"`;

exports[`Canonical ukchina frontPage SEO Twitter title 1`] = `"主页 - BBC 英伦网"`;
